dev/ProgressRing/ProgressRing.h

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+#include "common.h"
+
+#include "ProgressRing.g.h"
+#include "ProgressRing.properties.h"
+
+class ProgressRing :
+    public ReferenceTracker<ProgressRing, winrt::implementation::ProgressRingT>,
+    public ProgressRingProperties
+{
+
+public:
+    ProgressRing();
+    ~ProgressRing() {}
+
+    // IFrameworkElement
+    void OnApplyTemplate();
+
+    void OnPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args);
```

Comment:
Maybe remove?

Reply:
This is still unused it looks like.

---
In reply to: [346073791](https://github.com/microsoft/microsoft-ui-xaml/pull/1608#discussion_r346073791) [](ancestors = 346073791)